code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -438,12 +438,14 @@
 			SSjob.prioritized_jobs -= prioritized_job
 	dat += "<table><tr><td valign='top'>"
 	var/column_counter = 0
-	for(var/list/category in list(GLOB.command_positions) + list(GLOB.engineering_positions) + list(GLOB.supply_positions) + list(GLOB.nonhuman_positions - "pAI") + list(GLOB.civilian_positions) + list(GLOB.medical_positions) + list(GLOB.science_positions) + list(GLOB.security_positions))

```

Comment:
You're also changing the list used by silicons here.

Reply:
Changed the silicon category to use the same list definition that was previously used.